### ISSUE TITLE
exclusiveMinimum does not work using open api 3

### ISSUE THREAD

**Issue by asarver** [kogosoftwarellc/open-api#696]
json schema draft 4 says exclusiveMinimum should be a boolean. open api supports this field.
however, when specifying exclusiveMinimum as a boolean, there is an ajv error saying it should be a number.
if I make exclusiveMinimum a number, I get an error saying it should be a boolean.

using express-openapi 7.0.2
using node 12

When making it a number:
```js
const insertTestSchema = {
    type: 'object',
    properties: {
      cost: {
        type: 'number',
        exclusiveMinimum: 0
      }
    },
    required: ['cost']
  };

const apiDoc = {
    openapi: '3.0.0',
    info: {
        title: 'test',
        version: '1.0.0'
    },
    components: {
        schemas: {
            InsertTestSchema: insertTestSchema,
            Error: {
                additionalProperties: true
            }
        }
    },
    paths: {}
}

module.exports = apiDoc;
```

error:
```shell
> node index.js
express-openapi: Validating schema before populating paths
express-openapi: validation errors [
  {
    "keyword": "type",
    "dataPath": ".components.schemas['InsertJobSchema'].properties['cost'].exclusiveMinimum",
    "schemaPath": "http://json-schema.org/draft-04/schema#/properties/exclusiveMinimum/type",
    "params": {
      "type": "boolean"
    },
    "message": "should be boolean"
  }
  ....
```

when making it a boolean:
```js
const insertTestSchema = {
    type: 'object',
    properties: {
      cost: {
        type: 'number',
        exclusiveMinimum: true,
        minimum: 0
      }
    },
    required: ['cost']
  };

const apiDoc = {
    openapi: '3.0.0',
    info: {
        title: 'test',
        version: '1.0.0'
    },
    components: {
        schemas: {
            InsertTestSchema: insertTestSchema,
            Error: {
                additionalProperties: true
            }
        }
    },
    paths: {}
}

module.exports = apiDoc;
```

```shell
> node index.js 
../node_modules/ajv/lib/ajv.js:178
    else throw new Error(message);
         ^

Error: schema is invalid: data.properties['cost'].exclusiveMinimum should be number
    at Ajv.validateSchema (../node_modules/ajv/lib/ajv.js:178:16)
    at Ajv._addSchema (../node_modules/ajv/lib/ajv.js:307:10)
    at Ajv.addSchema (../node_modules/ajv/lib/ajv.js:137:29)
    at ../node_modules/openapi-request-validator/dist/index.js:106:19
    at Array.forEach (<anonymous>)
    at new OpenAPIRequestValidator (../node_modules/openapi-request-validator/dist/index.js:105:48)
    at ../node_modules/openapi-framework/dist/index.js:294:56
    at Set.forEach (<anonymous>)
    at ../node_modules/openapi-framework/dist/index.js:226:100
    at Array.forEach (<anonymous>)
```

**Comment by jsdevel** [comment]
please submit a pr

**Comment by asarver** [comment]
after doing some more research, it looks like this is an ajv v6 issue, which they're no longer actively developing.

since exclusiveMinimum is a supported field in the json draft schema and on the open api schema (https://swagger.io/specification/#schema-object), the property would need to be overridden in the open api schema validator, and probably the same for exclusiveMaximum too (haven't tried that one yet)

looking at the code, it looks like this will need to be handled in openapi-request-validator?

**Comment by asarver** [comment]
upgrading to the latest AJV version this is fixed (issue #727)
